.travis.yml

```diff
@@ -36,9 +44,23 @@ before_script:
   - sh -c 'if [ "$SYMFONY_VERSION" != "" ]; then composer require -n --no-update symfony/symfony=$SYMFONY_VERSION; fi;'
   - composer install --prefer-source
 
-  - sudo apt-get update
-  - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5
-  - sudo sed -i -e "s,/var/www,$(pwd)/Tests/Resources/public,g" /etc/apache2/sites-available/default
-  - sudo /etc/init.d/apache2 restart
+  - sudo chmod -R 0777 Tests/logs
+  - sudo apt-get update > /dev/null
```

Comment:
no need to use ``> /dev/null``. Travis is collapsing logs, so removing the output entirely only makes debugging harder while not increasing the readability of the build